### PR TITLE
Add auto starting for new processor group

### PR DESCRIPTION
**Pull request by ranum**
This change adds the automatic start for the new clinical ingestion processor group that was previously added by the script.

### CODE REVIEW

**Review by ibmatc — APPROVED** [review]
LGTM

**Review by rguderian — APPROVED** [review]

**Comment by ranum** [comment]
This new pull requests makes sure that all the processors actually start up by polling the stopped count for each processor group.  The processor groups are started from the "bottom up".  Finally, I had forgotten to bump the chart version which is now passing thanks to the fix for the test.

File: clinical-ingestion/utilities/setupClinicalIngestionFlow.py [hunk 1]
```diff
@@ -280,6 +284,58 @@ def main():
...
+        print(finalGroupList)
+
+    for thegroup in finalGroupList:
+        status2GetEndpoint = "nifi-api/process-groups/" + thegroup
+        resp = requests.get(url=baseURL + status2GetEndpoint)
+        statusdict = dict(resp.json())
+        stoppedcount = statusdict["stoppedCount"]
+        if debug:
+            print("Stopped Count: ", thegroup, "is ", int(stoppedcount))
+        while stoppedcount > 0:  #there are still some processors that are not running so start again
```
Comment (ibmatc) [inline clinical-ingestion/utilities/setupClinicalIngestionFlow.py:309]: do we need some sort of a max retry attempts count to prevent an infinite loop?

**Review by ibmatc — APPROVED** [review]
LGTM

File: clinical-ingestion/utilities/setupClinicalIngestionFlow.py (hunk 1 above)
Comment (ranum) [reply clinical-ingestion/utilities/setupClinicalIngestionFlow.py:309]: Good idea...didn't think of that but I suppose with nifi, it could happen.  Added.

**Pull request merged**